example/subscriberTest/testsubmain.go

```diff
@@ -113,6 +113,3 @@ func main() {
 	})
 	r.Run(":8086")
 }
-
-//b88847c0-a1ad-450e-a033-568e3a6cd4bc
```

Comment:
Only required changes should be shown in the PR. Others please revert.

Reply:
It was just a clean up